### PR TITLE
Add support for inclusions of files as text

### PR DESCRIPTION
**Pull request by ruippeixotog**

### CODE REVIEW

File: lib/frise/loader.rb [hunk 1]
```diff
@@ -36,18 +43,34 @@ def load(config_file, global_vars = {})
...
+      # process $content_include directives
+      config, content_include_confs = extract_content_include(config)
+      unless content_include_confs.empty?
+        raise "A #{@content_include_sym} must not have any sibling key" unless config.empty?
+
+        content = ''
+        content_include_confs.each do |include_conf|
+          extra_vars = (include_conf['vars'] || {}).map { |k, v| [k, root_config.dig(*v.split('.'))] }.to_h
+          extra_consts = include_conf['constants'] || {}
+          symbol_table = merge_at(root_config, at_path, config).merge(global_vars).merge(extra_vars).merge(extra_consts)
```
Comment (jcazevedo) [inline lib/frise/loader.rb]: The logic for building the symbol table for `$include` and `$content_include` is very similar. Can we extract this to a method?

File: lib/frise/loader.rb [hunk 2]
```diff
@@ -36,18 +43,34 @@ def load(config_file, global_vars = {})
...
+          content += Parser.parse_as_text(include_conf['file'], symbol_table)
```
Comment (jcazevedo) [inline lib/frise/loader.rb]: If the file doesn't exist we have a `no implicit conversion of nil into String` error here. Perhaps we can have a more specific error message.

File: lib/frise/loader.rb (hunk 1 above)
Comment (ruippeixotog) [reply lib/frise/loader.rb]: Sure, changed in 9200e51.

File: lib/frise/loader.rb (hunk 2 above)
Comment (ruippeixotog) [reply lib/frise/loader.rb]: The behavior most consistent with the renaming parts of Frise is to treat non-existing files as empty content (in this case, an empty string). Fixed in 07a1447.

**Pull request merged**